### ISSUE TITLE
Count Down Timers (Off Delay) does not work as expected

### ISSUE THREAD

**Issue by marekorok** [jneilliii/OctoPrint-Tasmota#89]
Hello,
would like to reopen #63 as seems does not work properly.
I would also first shutdown rpi gracefully, not killing it.

I do not give any delay and Power off is executed immediately.

Not sure if it's related to Tasmota version or else, I'm using 8.4.0

While testing found that plugin send 2 commands to Tasmota:
Delay 1200
Power Off

With such sequence Tasmota does not give any delay, Power off is executed immediately

```
16:58:10 CMD: delay 1200
16:58:10 MQT: stat/TasmotaOctoPrint/RESULT = {"Delay":1200}
16:58:12 CMD: power off
16:58:12 MQT: stat/TasmotaOctoPrint/RESULT = {"POWER1":"OFF"}
16:58:12 MQT: stat/TasmotaOctoPrint/POWER1 = OFF
```

Tested it manually through Tasmota console and confirmed the same behavior.

The proper way which works is using backlog command:
**backlog; Delay 1200; Power Off**

```
17:15:22 CMD: backlog; Delay 1200; Power Off
17:15:22 MQT: stat/TasmotaOctoPrint/RESULT = {"Delay":1200}
17:17:22 MQT: stat/TasmotaOctoPrint/RESULT = {"POWER1":"OFF"}
17:17:22 MQT: stat/TasmotaOctoPrint/POWER1 = OFF
```

Can this be fixed?

**Comment by jneilliii** [comment]
That's an interesting point, that is what I'm using assuming with use countdown timers.

https://github.com/jneilliii/OctoPrint-Tasmota/blob/223d2bb23fa62a71f010c39b025756a07c799db5/octoprint_tasmota/__init__.py#L347

**Comment by jneilliii** [comment]
I just upgraded my Tasmota to 8.5.1 (previously on 8.1.0) and tested and it does work as expected with the countdown timers option enabled. I think what you are seeing is the result of the backlog command, but not the actualy backlog command.

![image](https://user-images.githubusercontent.com/5249455/94943981-8b1dcb80-04a6-11eb-9ba6-89e8c5b83595.png)

**Comment by jneilliii** [comment]
You'll also notice in your own console results that there is a 2 minute delay between the delay command and the off command...

```
17:15:22 MQT: stat/TasmotaOctoPrint/RESULT = {"Delay":1200}
17:17:22 MQT: stat/TasmotaOctoPrint/RESULT = {"POWER1":"OFF"}
```

**Comment by marekorok** [comment]
Yes,

I have shown how it supposed to work, that was manual command on Tasmota console.
From Tasmota plugin of Octoprint I got wrong, not working sequence.

But, I have fixed it by myself :-) removed the plugin, cleaned files and reinstalled it again

![image](https://user-images.githubusercontent.com/28598034/94956919-394c6400-04ed-11eb-8e1d-75f95a04874f.png)

It's working now, sorry for bothering, looks like that some script stuck with some older version.
Should have done it before opening ticket, sorry for bothering and thank you for the great plugin!
